src/components/autocomplete/autocomplete.spec.js

```diff
@@ -27,6 +27,14 @@ describe('<md-autocomplete>', function() {
     return scope;
   }
 
+  function waitForVirtualRepeat(element) {
+    inject(function ($$rAF) {
+      $$rAF.flush();
+      element.scope().$apply();
+      $$rAF.flush();
+    });
```

Comment:
as @gkalpak says, the code is 100% self-documenting and not confusing at all.

However, I've added some comments as well.